sdk/core/azure-core/CMakeLists.txt

```diff
@@ -116,10 +116,9 @@ create_code_coverage(core azure-core azure-core-test)
 
 target_link_libraries(azure-core INTERFACE Threads::Threads)
 
-if(MSVC)
+if(WIN32)
     target_link_libraries(azure-core PRIVATE crypt32)
 else()
-    message("Find the OpenSSL package on MacOS, used for Base64 encoding.")
```

Comment:
Why was this removed?

Reply:
Because it would print on Linux as well.
And, cmake would be "talking" too much. We don't do this for any other package.